### ISSUE TITLE
Errors if back ticks (code) within link label

### ISSUE THREAD

**Issue by sarahjkim** [albert12132/templar#24]
**Markdown that caused the error**:

    - [Stack Overflow on `git reset` in plain English](http://stackoverflow.com/questions/2530060/can-you-explain-what-git-reset-does-in-plain-english)

**Error message**:
![screen shot 2015-01-11 at 6 53 02 pm](https://cloud.githubusercontent.com/assets/4628853/5698465/1b728914-99c3-11e4-9d90-0258f25234c4.png)

**Markdown that does not error**:

    - [Stack Overflow on git reset in plain English](http://stackoverflow.com/questions/2530060/can-you-explain-what-git-reset-does-in-plain-english)

**Comment by albert12132** [comment]
This behavior is undefined by the original Markdown spec. Does this situation happen often for you? If so, I can add support for this.

**Comment by sarahjkim** [comment]
It's happened a couple times, so support would be nice! Btw, 61B is using templar and loving it!

Basically everything, including guides are being generated with templar. I wrote a tiny little script called generate.py. I'll show it to you sometime.
http://berkeley-cs61b.github.io/public_html/assign.html
http://berkeley-cs61b.github.io/public_html/materials/guides/using-git.html

**Comment by albert12132** [comment]
Should be fixed in v1.3.1. Let me know if there are any other issues.
